File: Server/team05_server_source/Server/src/main/java/de/team05/server/json/MatchFinishJson.java
package de.team05.server.json;

public class MatchFinishJson extends LogPayloadJson {
    Integer endRound;
    Integer leftPoints;
    Integer rightPoints;
    String winnerUserName;
    String victoryReason;

    public MatchFinishJson(Integer endRound, Integer leftPoints, Integer rightPoints, String winnerUserName, String victoryReason) {
        this.endRound = endRound;
        this.leftPoints = leftPoints;
        this.rightPoints = rightPoints;
        this.winnerUserName = winnerUserName;
        this.victoryReason = victoryReason;
    }
}

File: Client/Client/Assets/Scripts/JsonScripts/MatchFinishJson.cs
using System;

[Serializable]
public class MatchFinishJson
{
    public int endRound;
    public int leftPoints;
    public int rightPoints;
    public string winnerUserName;
    public string victoryReason;

    //public MatchFinishJson(int endRound, int leftPoints, int rightPoints, string winnerUserName, string victoryReason)
    //{
    //    this.endRound = endRound;
    //    this.leftPoints = leftPoints;
    //    this.rightPoints = rightPoints;
    //    this.winnerUserName = winnerUserName;
    //    this.victoryReason = victoryReason;
    //}
}